### ISSUE TITLE
Is it possible to disable some configs routes and presets?

### ISSUE THREAD

**Issue by GustavoFenilli** [litstack/litstack#99]
What if I don't need permissions, how would I disable the preseted nav and links created by the package? The package could have a Lit::disableRoutes(); to use inside the RouteServiceProvider, or at least a way to modify parameters, so a sugestion would be include all Config files that are used inside the publish, so they could be overwritten.

**Comment by cbl** [comment]
Hey, this is actually a good Idea to exclude the default config Routes. Could you tell me your exact usecase?

I would think of something in the Serviceprovider saying:

```php
Lit::loadCrudRoutesFrom(...);
```

**Comment by GustavoFenilli** [comment]
The thing is, right now the permissions and the logins aren't easy to modify, I didn't see a way to modify the route prefix for it like the profile or user, what if I want to make my login page using livewire?

Also for me I would not need permissions on my application, because its the admin and only the admin that will use it, so routes and nav presets aren't needed.

There is also the problem with the default views sometimes havings the modals without localization using __lit().

So basically I would say that anything that is a default config and the user can see, should be able to change, adding to the publish would make it easier to extend or modify both Config and Views.

**Comment by cbl** [comment]
In the pr https://github.com/litstack/litstack/pull/90 I added support for custom authorizations. This was tagged in [v3.1.2](https://github.com/litstack/litstack/releases/tag/v3.1.2). So it is possible to write your own authorization/login page. However I haven't documented this yet.

You are right, every service is provided by default in the `Ignite\Application\ApplicationServiceProvider` is not a good thing. It would be better to move this to the configuration file so you could disable services if they are not needed, I will definitely implement that. Also the login route should be overwritable.

At the moment you are able to edit the user config and the profile settings config in `lit/app/Config/User` and you could just delete them if you don't need them. The only thing that cannot be edited is the permission page.

**Comment by GustavoFenilli** [comment]
I didn't know about the custom auths cool addition ( not gonna use, I like the ones right now, it was just a sugestion.

I liked the idea of passing those providers to the configuration file, it would be a good thing, i will be waiting for that PR.

**Comment by cbl** [comment]
The providers are now located in the `lit` config, however you may only remove the PermissionServiceProvider since the others are all required to use the litstack crud system.
